django_app/redbox_app/templates/accessibility-statement.html

```diff
@@ -1,46 +1,39 @@
-{% import "macros.html" as macros %}
-{% extends "base_generic_gov.html" %}
+{% extends "base.html" %}
 
-<!--TODO: Update system url and other variables-->
-
-{% block title %}
-  <title>Accessibility statement - Redbox Copilot - GOV.UK</title>
-{% endblock %}
+{% set pageTitle = "Accessibility statement" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot Platform</h1>
-  <div class="govuk-body">
-    <p>This accessibility statement applies to Redbox Copilot - <a href="<SYSTEM_URL>" class="govuk-link"><SYSTEM_URL></a>.</p>
-    <p>This website is run by the Cabinet Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
+  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot</h1>
+
+    <p class="govuk-body">This accessibility statement applies to Redbox Copilot.</p>
```

Comment:
You should only have to add `govuk-body` to the `body` tag in `base.html`

Reply:
Yes, true, but strangely the spacing was more condensed when I did that. I'll take another look as it'd be tidier to only have it once.

Reply:
Because `.govuk-body` has a margin-bottom set on it, this will be missed if it's only used on a container element. That explains why it's needed for the spacing. Though there will be contexts where it'll be okay, I don't believe this is one. There are also styles for heading elements immediately preceeded by `.govuk-body`.
Looking on gov.uk there are examples of `<p>` elements without `govvuk-body`.